projects/fizzbuzz/java/README.md

```diff
@@ -1 +1,8 @@
-Place java solutions to fizz buzz in this directory
+Author John Watts

```

Comment:
The .md file extension is for markdown files. [Github offers its own variant](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf) of markdown formatting that you can reference. 

[Sublime Text 3 ](https://www.sublimetext.com/3) has a package control that allows for additional features to be added. There is a package you can add called MarkdownPreview that will allow you to preview your README.md files before pushing them to Github.